server/Resources/Resources.resx

```diff
@@ -4848,6 +4851,9 @@ Set to "External URL" to use an API definition that is hosted elsewhere.</value>
   <data name="githubOrgMessage" xml:space="preserve">
     <value>If you can't find an organization or repository, you might need to enable additional permissions on GitHub.</value>
   </data>
+  <data name="projectWithoutGitRepoNotFoundMessage" xml:space="preserve">
+    <value>If you can't find the project, there might be not any Git repository in the project.</value>
```

Comment:
Should we say... "Only projects with valid git repositories will be listed"?

You can take feedback from PM as well.

Reply:
Will definitely take PM/Sachin's feedback over here.